zabbix/check.py

```diff
@@ -0,0 +1,334 @@
+"""
+    StackState.
+    Zabbix host topology and problem integration
+
+    Current limitations:
+    - Unsupported Zabbix Problem acks
+    - Unsupported Zabbix version 4+ severity upgrades, currently the trigger's severity is used for compatibility
+"""
+
+import requests
+import logging
+import time
+
+from checks import AgentCheck, CheckException
+
+
+class ZabbixHost:
+    def __init__(self, hostid, host, name, host_groups):
+        assert(type(host_groups == list))
+        self.host_id = hostid
```

Comment:
will be good to have the same name for `hostid` as `host_id`.

Reply:
nice catch, simple copy/paste action of the Zabbix payload ended up in there. Fixed.